### PR TITLE
Update WebKit to r195440

### PR DESCRIPTION
**Pull request by Constellation**
- Support destructuring in catch heads
- Function is subclassible
- String is subclassible

### CODE REVIEW

**Comment by zloirock** [comment]
Please, fix duplicate properties and rebuild HTML.

**Comment by Constellation** [comment]
Oops, I missed it.

**Comment by Constellation** [comment]
Updated :)

File: data-es6.js [hunk 1]
```diff
@@ -13305,6 +13311,7 @@ exports.tests = [
         firefox45:   true,
         xs6:         true,
         ejs:         true,
+        webkit:      true,
```
Comment (zloirock) [inline data-es6.js]: ^^^ duplicated ^^^

Comment (Constellation) [inline data-es6.js]: Oops, thanks!

**Comment by Constellation** [comment]
OK, fixed.

**Pull request merged**